### PR TITLE
[LIQ-424] Sort orders

### PR DESCRIPTION
**Pull request by ahsan8244**
Screenshot 2022-04-08 at 2 44 52 AM

The photo shows an outdated screenshot, I made the selected label's text purple in the latest commit

### CODE REVIEW

File: src/core/Orders.tsx [hunk 1]
```diff
@@ -24,13 +56,17 @@ export const Orders: React.FC<OrdersProps> = ({
...
 }) => {
+  const [sortedByOption, setSortedByOption] = useState(SORT_OPTIONS[0]);
   const [orders, setOrders] = useState<any[]>([]);
   const [loading, setLoading] = useState(false);
 
   // handle fetch data
   useEffect(() => {
     setLoading(true);
-    fetchOrdersByStoreId(candyShop.candyShopAddress().toString())
+    fetchOrdersByStoreId(
```
Comment (ahsan8244) [inline src/core/Orders.tsx:66]: Why have we not been using `candyShop.orders()` here?

**Comment by ahsan8244** [comment]
Just a terminology nit, I have used `sort` instead of `order`. Because of our association of the word "orders" with NFT orders, it is probably best to use the word "sort" for sorting purposes.

File: src/core/Orders.tsx (hunk 1 above)
Comment (RustySol) [reply src/core/Orders.tsx:66]: Agree this we should use candyshop.orders()

File: src/api/backend/OrderAPI.ts [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+};
 
 export async function fetchOrdersByStoreId(
-  storeId: string
+  storeId: string,
+  sortBy?: SortBy
 ): Promise<ListBase<Order>> {
+  const queryObject = {} as any;
+  if (sortBy) {
+    queryObject['orderByArr'] = JSON.stringify(sortBy);
```
Comment (RustySol) [inline src/api/backend/OrderAPI.ts:16]: does `sortBy` need to be an array?

Comment (ahsan8244) [reply src/api/backend/OrderAPI.ts:16]: A bit confused about this. When I tried sending an array, the backend is unable to parse it. Seems like the only thing that works is an object

Comment (RustySol) [reply src/api/backend/OrderAPI.ts:16]: noted

**Review by RustySol — APPROVED** [review]

**Pull request merged**